src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,82 @@
+@import "scss/constants.scss";
+
+.ScheduleEvent {
+  display: flex;
+  position: absolute;
+  border-radius: 18px;
+  padding: 0 4px 0 12px;
+  cursor: pointer;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3f3d42;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+  margin-top: 0.25rem;
+
+  &:hover {
+    background-color: #4c494f;
+    z-index: z(navbar__schedule-event--hover);
+  }
+
+  &--users {
+    color: #4c494f;
+    background-color: #ebebeb;
+
+    &:hover {
+      background-color: white;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: black;
+    }
+  }
+
+  &--live {
+    background-color: #7c46fb;
+    color: white;
+
+    &:hover {
+      background-color: #8755fb;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: white;
+    }
+  }
```

Comment:
colors should be vars

Reply:
used vars for colors here: https://github.com/sparkletown/sparkle/pull/1302/commits/06620c6fb5ce10998508b5403d8b8cf907ed32d6
and here: https://github.com/sparkletown/sparkle/pull/1302/commits/4b5062d7063d480be64ec88429bf24b77fd196ff